packages/opensrp-plans/src/constants.tsx

```diff
@@ -37,3 +37,21 @@ export const SERVICE_POINT_CHECK = 'Service Point Check';
 
 /** Field to sort plans by */
 export const SORT_BY_EFFECTIVE_PERIOD_START_FIELD = 'date';
+
+// opensrp api strings
+export const OPENSRP_API_BASE_URL = 'https://mg-eusm-staging.smartregister.org/opensrp/rest/';
```

Comment:
Do we absolutely have to reference `mg-eusm-staging.smartregister.org` as the default value of this?  I would rather that was something more generic like the OpenSRP staging server.  `mg-eusm-staging.smartregister.org` can be supplied as an env when you deploy this for EUSM